src/components/Reservations/Reservation.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Reservation = (props) => {
+  const {
+    userId, roomId, startDate, endDate, city,
+  } = props;
+
```

Comment:
### Great job.to have a dedicated page to see a detail of a single reservation.  
- Though for code simplicity. Instead of passing reservation detail as `props` which requires validating all the `props ` and having more code, I suggest we only pass the `ID `of the reservation, Then accessing it from the reducer. That way, the data will be even more synchronous. 😎